### DISCUSSION TITLE
user-guide/faqs/plugins

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#183] [文档评论区]
# user-guide/faqs/plugins

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/faqs/plugins.html

**Comment by sanggrub** [comment]
Zetoro 7中 gpt插件，显示安装成功，但布显示，并且秘钥安完成后，回车，没有回应

**Comment by WngelSuyin** [comment]
请问Zotero7怎么加入ZotFile插件？ 官网的最新fix支持Zotero5 好像不兼容

**Comment by miduner22** [comment]
请问为什么zotero integration选择import命令后，zotero显示Better BibTex: CAYW pick failed？

**Comment by chdliu1974** [comment]
接入deepseek后应用askpdf 没有结果，提示如下，怎么解决？

**Reply by chdliu1974** [reply]
提示是这个：GPT版本是1.7.4
🪐AskPDF execution error: _0x254a78.publisher2models.get(...) is undefined

Meet.Zotero.getRelatedText(Meet.Global.input || "概括本文")

**Comment by VAMMGA** [comment]
请问下使用Zetoro 7的Translate插件，调用了大模型api，显示连接成功了，但不是正常翻译选取的内容，也没有提示报错，而是显示“您好 很高兴见到您 我是deepseek等等等”请问是什么原因呢

**Reply by VAMMGA** [reply]
现在是可以划词翻译 但是句子就不行，选中句子是提示没有选择内容，或者识别为一个句号或者识别成一个表情，识别不了句子

**Reply by VAMMGA** [reply]
使用的是maas平台提供的api密钥，确认api正常可用，不是用的谷歌的api
